lib/prometheus_exporter/server/puma_collector.rb

```diff
@@ -47,6 +48,15 @@ def metrics
     end
 
     def collect(obj)
+      now = ::Process.clock_gettime(::Process::CLOCK_MONOTONIC)
+
+      obj["created_at"] = now
+
+      @puma_metrics.delete_if do |current|
+        (obj["pid"] == current["pid"] && obj["hostname"] == current["hostname"]) ||
```

Comment:
I took a look at the puma collector and I don't think `pid` and `hostname` are included in the `obj`?

Reply:
@eviltrout you're right , seems we need to leave only current["created_at"] + MAX_PUMA_METRIC_AGE < now